database/migrations/2020_07_07_200824_create_ministries_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateMinistriesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('ministries', function (Blueprint $table) {
```

Comment:
ministry

Reply:
@goodhands  I thought the tables are usually in the plural form